CriticalMassTests/StringAdditionsTests.swift

```diff
@@ -0,0 +1,51 @@
+//
+// Created for CriticalMaps in 2020
+
+@testable import CriticalMaps
+import XCTest
+
+class StringAdditionsTests: XCTestCase {
+    func test_removeDatePatternShouldRemovePattern1() {
+        // given
+        let title = "Critical Mass Berlin"
+        let date = "28.02.2020"
+        let titleWithDate = "\(title) \(date)"
+        // when
+        let strippedTitle = titleWithDate.removedDatePattern()
+        // then
+        XCTAssertEqual(strippedTitle, title)
+    }
+
+    func test_removeDatePatternShouldRemovePattern2() {
+        // given
+        let title = "Critical Mass Berlin"
+        let date = "28/02/2020"
+        let titleWithDate = "\(title) \(date)"
+        // when
+        let strippedTitle = titleWithDate.removedDatePattern()
+        // then
+        XCTAssertEqual(strippedTitle, title)
+    }
+
+    func test_removeDatePatternShouldNotRemovePattern1() {
+        // given
+        let title = "Critical Mass Berlin"
+        let date = "02/28/20"
+        let titleWithDate = "\(title) \(date)"
+        // when
+        let strippedTitle = titleWithDate.removedDatePattern()
+        // then
+        XCTAssertNotEqual(strippedTitle, title)
```

Comment:
Hmm. Not sure about that since I tried to follow the swift naming convention for nonmutating functions. That is also the reason why I would make this a computed property.
https://swift.org/documentation/api-design-guidelines/#name-according-to-side-effects

Reply:
So the name should point out that the `titleWithDate` can actually not be modified. But maybe I could do a better job at naming things. Hardest part of programming. Again